src/source.css

```diff
@@ -5589,9 +5592,18 @@ ul.bda-slist {
 }
 
 /* Message hover toolbar */
-.message-2qnXI6.selected-2P5D_Z .buttons-cl5qTG,
-.mouse-mode .message-2qnXI6:hover .buttons-cl5qTG {
-  visibility: hidden !important;
+.wrapper-2aW0bm {
+  background: rgba(0, 0, 0, 0.4) !important;
+  border: 1px solid var(--hover-color) !important;
+}
+.button-1ZiXG9 {
+  color: var(--hover-bright-color) !important;
+}
+.button-1ZiXG9:hover {
```

Comment:
Let's also add `.button-1ZiXG9:active`, even though it's somewhat of an odd edge case in this scenario.
You can get this state by clicking a button without releasing and then dragging out of the bounds of the button.
An odd one, I know, but eh ¯\_(ツ)_/¯

Reply:
Which color would you think fits?

Reply:
`var(--hover-dark-color)` actually fits well, I'll add that to the PR

Reply:
I mean, it would fit if that action actually did anything 😂
We could also just use `--hover-color` to not bring any extra attention to it, if it happens.

Reply:
Also there's the `.button-1ZiXG9.selected-LCBEAU`, when the emoji tab is opened from the "Add Reaction" button